editor/src/components/editor/editor-component.tsx

```diff
@@ -504,18 +508,24 @@ export function EditorComponent(props: EditorProps) {
 
   const dispatch = useDispatch()
 
+  const roomId = React.useMemo(
+    () => (projectId == null ? generateUUID() : `project-room-${projectId}`),
```

Comment:
should / could this generateUUID() part be skipped and the RoomProvider just given a `null` (or any other set of props that makes the RoomProvider dormant / fail silently)?

Reply:
Frustratingly, the `id` field is a mandatory field of type `string`. I briefly considered optionally including it in the stack of providers based on a nullable `roomId` but was a bit dubious of what that might do to the editor and it would probably make `useRoom` break due to the `RoomProvider` not being available.

Reply:
nein, liveblocks needs an actual string :( :( :(

Reply:
ok thanks :)